app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,243 @@
+class ChargesController < ApplicationController
```

Comment:
Getting this error when running Rspec. How do I resolve?
<img width="874" alt="Screen Shot 2019-05-12 at 2 18 33 PM" src="https://user-images.githubusercontent.com/10746770/57587868-e7063f80-74c0-11e9-917d-2d90f1869ff7.png">

Reply:
You need to decrypt it by doing this: 
gpg --passphrase "getrekt" --decrypt --output config/application.yml config/application.yml.asc

Reply:
You shouldn't have put the password here....this is a public repo

Invalidate all API keys in this file, generate new ones, re-encrypt the file, and then securely send me the keys. Email me, I'll send you my GPG key.

Reply:
Sorry, my bad. I deleted the comments. Though the api key encrypted that I provided is just my test key that I personally created. I’ll do this now.